docs/docs/installation_guide/installation/pre-installation/instance-req.md

```diff
@@ -74,10 +74,15 @@ To add a role to an Amazon EC2 instance that is already running:
 ```
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
 source ~/.bashrc
-nvm install 12
-npm install -g serverless pnpm hygen
+nvm install v14.17.0
+npm install -g pnpm@5.18.9 serverless@3.8.0
```

Comment:
latest pnpm is 7.5.0, why pinning a quite old version here?

Reply:
also any reason to pin serverless framework?

Reply:
For pnpm, if we used 7.5.0, it will caused deployment error as below:

```
addons/addon-base-raas/packages/base-raas-cfn-templates
└─┬ eslint-config-airbnb
  ├── ✕ missing peer eslint-plugin-jsx-a11y@^6.2.3
  ├── ✕ missing peer eslint-plugin-react@^7.19.0
  └── ✕ missing peer eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"
Peer dependencies that should be installed:
  eslint-plugin-jsx-a11y@^6.2.3                 eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"  eslint-plugin-react@^7.19.0                   

addons/addon-base-raas/packages/base-raas-post-deployment
└─┬ eslint-config-airbnb
  ├── ✕ missing peer eslint-plugin-jsx-a11y@^6.2.3
  ├── ✕ missing peer eslint-plugin-react@^7.19.0
  └── ✕ missing peer eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"
Peer dependencies that should be installed:
  eslint-plugin-jsx-a11y@^6.2.3                 eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"  eslint-plugin-react@^7.19.0                   

addons/addon-base-raas/packages/base-raas-workflow-steps
└─┬ eslint-config-airbnb
  ├── ✕ missing peer eslint-plugin-jsx-a11y@^6.2.3
  ├── ✕ missing peer eslint-plugin-react@^7.19.0
  └── ✕ missing peer eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"
Peer dependencies that should be installed:
  eslint-plugin-jsx-a11y@^6.2.3                 eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"  eslint-plugin-react@^7.19.0                   

addons/addon-base-raas/packages/base-raas-workflows
└─┬ eslint-config-airbnb
  ├── ✕ missing peer eslint-plugin-jsx-a11y@^6.2.3
  ├── ✕ missing peer eslint-plugin-react@^7.19.0
  └── ✕ missing peer eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"
Peer dependencies that should be installed:
  eslint-plugin-jsx-a11y@^6.2.3                 eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"  eslint-plugin-react@^7.19.0                   

addons/addon-base-workflow-api/packages/base-worklfow-api
└─┬ eslint-config-airbnb
  ├── ✕ missing peer eslint-plugin-jsx-a11y@^6.2.3
  ├── ✕ missing peer eslint-plugin-react@^7.19.0
  └── ✕ missing peer eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"
Peer dependencies that should be installed:
  eslint-plugin-jsx-a11y@^6.2.3                 eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"  eslint-plugin-react@^7.19.0                   

addons/addon-base-workflow/packages/base-workflow-core
└─┬ eslint-config-airbnb
  ├── ✕ missing peer eslint-plugin-jsx-a11y@^6.2.3
  ├── ✕ missing peer eslint-plugin-react@^7.19.0
  └── ✕ missing peer eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"
Peer dependencies that should be installed:
  eslint-plugin-jsx-a11y@^6.2.3                 eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"  eslint-plugin-react@^7.19.0                   

addons/addon-base-workflow/packages/base-workflow-templates
└─┬ eslint-config-airbnb
  ├── ✕ missing peer eslint-plugin-jsx-a11y@^6.2.3
  ├── ✕ missing peer eslint-plugin-react@^7.19.0
  └── ✕ missing peer eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"
Peer dependencies that should be installed:
  eslint-plugin-jsx-a11y@^6.2.3                 eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"  eslint-plugin-react@^7.19.0                   

addons/addon-base-workflow/packages/base-worklfow-steps
└─┬ eslint-config-airbnb
  ├── ✕ missing peer eslint-plugin-jsx-a11y@^6.2.3
  ├── ✕ missing peer eslint-plugin-react@^7.19.0
  └── ✕ missing peer eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"
Peer dependencies that should be installed:
  eslint-plugin-jsx-a11y@^6.2.3                 eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"  eslint-plugin-react@^7.19.0                   

addons/addon-base-workflow/packages/workflow-engine
└─┬ eslint-config-airbnb
  ├── ✕ missing peer eslint-plugin-jsx-a11y@^6.2.3



main/packages/services
└─┬ eslint-config-airbnb
  ├── ✕ missing peer eslint-plugin-jsx-a11y@^6.2.3
  ├── ✕ missing peer eslint-plugin-react@^7.19.0
  └── ✕ missing peer eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"
Peer dependencies that should be installed:
  eslint-plugin-jsx-a11y@^6.2.3                 eslint-plugin-react-hooks@"^2.5.0 || ^1.7.0"  eslint-plugin-react@^7.19.0                   

main/solution/backend
└─┬ serverless-webpack
  └─┬ ts-node
    └── ✕ missing peer typescript@>=2.7
Peer dependencies that should be installed:
  typescript@>=2.7  

main/solution/post-deployment
└─┬ serverless-webpack
  └─┬ ts-node
    └── ✕ missing peer typescript@>=2.7
Peer dependencies that should be installed:
  typescript@>=2.7  

main/solution/pre-deployment
├─┬ serverless
│ └─┬ @serverless/components
│   └─┬ inquirer-autocomplete-prompt
│     └── ✕ missing peer inquirer@"^5.0.0 || ^6.0.0 || ^7.0.0"
└─┬ serverless-webpack
  └─┬ ts-node
    └── ✕ missing peer typescript@>=2.7
Peer dependencies that should be installed:
  inquirer@"^5.0.0 || ^6.0.0 || ^7.0.0"  typescript@>=2.7                       

main/solution/prepare-devops-acc
└─┬ serverless
  └─┬ @serverless/components
    └─┬ inquirer-autocomplete-prompt
      └── ✕ missing peer inquirer@"^5.0.0 || ^6.0.0 || ^7.0.0"
Peer dependencies that should be installed:
  inquirer@"^5.0.0 || ^6.0.0 || ^7.0.0"  

main/solution/ui
├─┬ react-scripts
│ ├── ✕ missing peer typescript@^3.2.1
│ ├─┬ @typescript-eslint/eslint-plugin
│ │ ├── ✕ missing peer typescript@"*"
│ │ └─┬ @typescript-eslint/experimental-utils
│ │   └─┬ @typescript-eslint/typescript-estree
│ │     ├── ✕ missing peer typescript@"*"
│ │     └─┬ tsutils
│ │       └── ✕ missing peer typescript@">=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta"
│ ├─┬ @typescript-eslint/parser
│ │ └── ✕ missing peer typescript@"*"
│ ├─┬ eslint-config-react-app
│ │ └── ✕ missing peer typescript@"*"
│ └─┬ react-dev-utils
│   └─┬ fork-ts-checker-webpack-plugin
│     └── ✕ missing peer typescript@">= 2.7"
├─┬ ts-pnp
│ └── ✕ missing peer typescript@"*"
└─┬ serverless
  └─┬ @serverless/components
    └─┬ inquirer-autocomplete-prompt
      └── ✕ missing peer inquirer@"^5.0.0 || ^6.0.0 || ^7.0.0"  
```

After discussion in the swb slack channel, they suggest to use a fixed version, they recommend to use v5.18.9.

For serverless, they also suggest to use 3.8.0, but I just verified again, we can use 3.19.0(latest version).
Refactored the doc.